integration/fixtures/server-ca-csr-wildcard.json

```diff
@@ -0,0 +1,21 @@
+{
+  "key": {
+    "algo": "rsa",
+    "size": 4096
+  },
+  "names": [
+    {
+      "O": "etcd",
+      "OU": "etcd Security",
+      "L": "San Francisco",
+      "ST": "California",
+      "C": "USA"
+    }
+  ],
+  "CN": "example.com",
+  "hosts": [
+    "*.etcd.local",
+    "127.0.0.1",
```

Comment:
the one I forgot to include in the original patch (oops) has this:
```
  "hosts": [
    "*.etcd.local"
  ]
```
which skips the IP SAN check and forces the DNS SAN path